akka-projection-dynamodb/src/main/resources/reference.conf

```
@@ -22,6 +22,19 @@ akka.projection.dynamodb {
 
     # Trying to batch insert offsets in batches of this size.
     offset-batch-size = 20
+
+    # Number of slices (within a given projection's slice range) which will be queried for
+    # offsets simultaneously.  The underlying Dynamo client must be able to handle
+    # (`http.max-concurrency` plus `http.max-pending-connection-acquires`) at least this number
+    # of concurrent requests.
+    #
+    # Set to 1024 to always query for all slices simultaneously.  The minimum allowed value
+    # is 1.  If there are more than 64 slices in a range (e.g. fewer than 16 projections
+    # consuming events), then increasing this may result in slightly faster projection starts;
+    # conversely, if there are many slices being projected using a given Dynamo client,
+    # reducing this may result in fewer restarts of the projection due to failure to query
+    # starting offsets.
+	offset-slice-read-parallelism = 64
```

Currently a tab, rather than spaces.

```suggestion
    offset-slice-read-parallelism = 64
```